src/main/java/seedu/address/logic/commands/AddEntryCommand.java

```diff
@@ -0,0 +1,64 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_END_DATE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_START_DATE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.entry.Entry;
+
+/**
+ * Adds an Entry to the list
+ */
+public class AddEntryCommand extends Command {
+
+    public static final String COMMAND_WORD = "eadd";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds an entry to the list. "
+            + "Parameters: "
+            + PREFIX_NAME + "NAME "
+            + "[" + PREFIX_START_DATE + "START DATE] "
+            + PREFIX_END_DATE + "END DATE "
+            + "[" + PREFIX_TAG + "TAG]...\n"
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_NAME + "consultation 1 "
+            + PREFIX_START_DATE + "2021-04-01 17:00 "
+            + PREFIX_END_DATE + "2021-04-01 19:00 "
+            + PREFIX_TAG + "CS2100";
+
+    public static final String MESSAGE_SUCCESS = "New entry added: %1$s";
```

Comment:
should we move these messages to commons.core.Messages? similar approach can be used for #94 #95